include/deal.II/lac/trilinos_sparse_matrix.h

```diff
@@ -2403,7 +2403,7 @@ namespace TrilinosWrappers
          * Epetra_Operator.
          *
          * @note This function will only be operable if the payload has been
-         * initalised with an InverseOperator, or is a wrapper to a preconditioner.
+         * initialised with an InverseOperator, or is a wrapper to a preconditioner.
```

Comment:
would you mind making this `initialized` ?

Reply:
No problem. Will do that and repush  soon